packages/graphql-hooks/src/GraphQLClient.js

```diff
@@ -15,6 +15,29 @@ class GraphQLClient {
       this.subscriptionClient = this.subscriptionClient()
     }
 
+    this.verifyConfig(config)
+
+    this.cache = config.cache
+    this.headers = config.headers || {}
+    this.ssrMode = config.ssrMode
+    this.ssrPromises = []
+    this.url = config.url
+    this.fetch =
+      config.fetch || (typeof fetch !== 'undefined' && fetch && fetch.bind())
+    this.fetchOptions = config.fetchOptions || {}
+    this.FormData =
+      config.FormData ||
+      (typeof FormData !== 'undefined' ? FormData : undefined)
+    this.logErrors = config.logErrors !== undefined ? config.logErrors : true
+    this.onError = config.onError
+    this.useGETForQueries = config.useGETForQueries === true
+    this.middleware = new Middleware(config.middleware || [])
+
+    this.mutationsEmitter = new EventEmitter()
+  }
+
+  /** Checks that the given config has the correct required options */
+  verifyConfig(config) {
```

Comment:
not sure I understand what happened here. has the code only been moved around? please clarify because it's not obvious by just reading the code

Reply:
Yes, just moved around so that the local version of the client can override this function to remove all those checks